<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recApp</class>
 <widget class="QWidget" name="recApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>recApp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>rec.ico</normaloff>rec.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>stop </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
